docs/scripts/schemas/unversioned/eas-json-ios-schema.js

```diff
@@ -0,0 +1,173 @@
+    ]
+  },
+  {
+    name: 'autoIncrement',
+    type: 'boolean | \"version\" | \"buildNumber\"',
+    description: [
+      'Controls how EAS CLI bumps your application build version. Defaults to `false`',
+      ' - `"version"` - the patch of `expo.version` is bumped (e.g. `1.2.3` -> `1.2.4`).',
+      ' - `"buildNumber"` (or `true`) - the last component of `expo.ios.buildNumber` is bumped (e.g. `1.2.3.39` -> `1.2.3.40`).',
+      ' - `false` - versions won\'t be bumped automatically',
+      '',
+      'Versions will also be updated in native code if it\'s a bare project. `expo.version` corresponds to `CFBundleShortVersionString` and `expo.ios.buildNumber` to `CFBundleVersion` in the `Info.plist`. The App Store is using those values to identify the app build, `CFBundleShortVersionString` is the version visible to users, whereas `CFBundleVersion` defines the build number. The combination of those needs to be unique, so you can bump either of them.',
+    ],
+  },
+  {
+    name: 'image',
+    type: 'string',
+    description: [
+      'Image with build environment. [Learn more about it here](../../build-reference/infrastructure).',
+    ],
+  },
+  {
+    name: 'node',
+    type: 'string',
+    description: [ 'Version of Node.js.' ],
+  },
+  {
+    name: 'yarn',
+    type: 'string',
+    description: [ 'Version of Yarn.' ],
+  },
+  {
+    name: 'bundler',
+    type: 'string',
+    description: [ 'Version of [bundler](https://bundler.io/).' ],
+  },
+  {
+    name: 'fastlane',
+    type: 'string',
+    description: [ 'Version of fastlane.' ],
+  },
+  {
+    name: 'cocoapods',
+    type: 'string',
+    description: [ 'Version of CocoaPods.' ],
+  },
+  {
+    name: 'expoCli',
+    type: 'string',
+    description: [
+      'Version of [expo-cli](https://www.npmjs.com/package/expo-cli) used to [prebuild](../../workflow/expo-cli/#expo-prebuild) your app. It does not affect bare projects.',
+    ],
+  },
+  {
+    name: 'env',
+    type: 'object',
+    description: [
+      'Environment variables that should be set during the build process (should only be used for values that you would commit to your git repository, i.e. not passwords or secrets).',
+    ],
+  },
+  {
+    name: 'cache',
+    type: 'object',
+    description: [
+      'Configuration for caching machanism. This feature is intended for caching values that require a lot of computation, e.g. compilation results (both final binaries and any intermediate files), but it wouldn\'t work well for `node_modules` because the cache is not local to the machine, so a download speed is similar to downloading from the npm registry.'
```

Comment:
```suggestion
      'Cache configuration. This feature is intended for caching values that require a lot of computation, e.g. compilation results (both final binaries and any intermediate files), but it wouldn\'t work well for `node_modules` because the cache is not local to the machine, so the download speed is similar to downloading from the npm registry.'
```